src/binarytree.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+export default class binarySearchTree {
+  constructor() {
+    this.root = null;
+  }
+
+  contains(value) {
```

Comment:
this looks like it wont compile. There is a closing parenthesis on line 10, which seems out of place